src/DocRaptor.php

```diff
@@ -176,26 +188,36 @@ public function fetchDocument($filename = false)
             }
 
             if (!empty($this->document_content)) {
-                $fields['doc[document_content]'] = urlencode($this->document_content);
+                $fields['doc[document_content]'] = $this->document_content;
             } else {
-                $fields['doc[document_url]'] = urlencode($this->document_url);
+                $fields['doc[document_url]'] = $this->document_url;
             }
 
-            $fields_string = http_build_query($fields);
+            $queryString = http_build_query($fields);
```

Comment:
I'm good with that. I just couldn't remember if idiomatic PHP was camel or snake.